pint/registry.py

```diff
@@ -216,7 +219,7 @@ def __init__(
         self.auto_reduce_dimensions = auto_reduce_dimensions
 
         #: Default locale identifier string, used when calling format_babel without explicit locale.
-        self.fmt_locale = self.set_fmt_locale(fmt_locale)
+        self.set_fmt_locale(fmt_locale)
```

Comment:
Changes to this file are independent of this PR. Have you considered using pylint to detect calling this function that doesn't return any value.

Reply:
They are independent but this was a bug that need to be fixed to make this PR work.